tests/benchdnn/graph/graph.cpp

```diff
@@ -474,6 +474,15 @@ int skip_unimplemented_partitions(const std::vector<partition> &partitions,
     BENCHDNN_PRINT(3, "[INFO]: n_partitions:%zd; ops_in_partitions:%s\n",
             partitions.size(), verbose_partitions_n_ops(partitions).c_str());
 
+    // As softmax inf_as_zero algorithm is unsupported on NV platform now, we
+    // skip all the SDPA cases in benchdnn graph.
+    if (is_nvidia_gpu()) {
+        if (dg.get_recognized_pattern() == graph_recognized_pattern_t::sdpa) {
```

Comment:
```suggestion
    if (is_nvidia_gpu() && dg.get_recognized_pattern() == graph_recognized_pattern_t::sdpa_fwd) {
```
The recognized_pattern is changed from sdpa to sdpa_fwd in the latest main.